src/modules/verifiableCredentials/index.ts

```diff
@@ -0,0 +1,27 @@
+export * from './verifiableCredentials.types';
+export * from './verifiableCredentialsBase.service';
+
+import { SignerService } from '../signer';
+import { VerifiableCredentialsServiceBase } from './verifiableCredentialsBase.service';
+
+export const getVerifiableCredentialsService = async (
+  signerService: SignerService
+): Promise<VerifiableCredentialsServiceBase> => {
+  const isNode =
```

Comment:
you can use https://github.com/energywebfoundation/iam-client-lib/blob/c45a81c36af782cc8d93d3e943e331789e66c571/src/utils/detectEnvironment.ts#L6